packages/frontend-common/formats/chart/network/useColorOverrides.tsx

```diff
@@ -1,56 +1,87 @@
 import { useMemo } from 'react';
 import { useTranslate } from '../../../i18n/I18NContext';
 
+export const matchOneOf = (patterns: string[], content: string) => {
+    if (!patterns || patterns.length === 0) {
+        return false;
+    }
+    return patterns.some((pattern) => {
+        return new RegExp(`^${pattern.replaceAll('*', '.*')}`, 'i').test(
```

Comment:
Maybe a dollar in the regex ? Otherwise this wildcard `a*e` could match 'abcdefg`

```suggestion
        return new RegExp(`^${pattern.replaceAll('*', '.*')}$`, 'i').test(
```